modules/ROOT/pages/functions/templates/node.adoc

```diff
@@ -0,0 +1,267 @@
+Node.js Function Template Reference
+===================================
+
+The Node.js function templates are designed to allow developers to concentrate
+solely on the business logic of the application, leaving things like managing
+network connections to the runtime framework. Whether responding to
+`CloudEvents` or simple HTTP requests, the function templates are simple and
+straightforward.
+
+== Function Invocation
+
+=== Parameters
+OpenShift Serverless functions are invoked when an incoming HTTP request is
+received. Invocation parameters include a `Context` object as the first
```

Comment:
I'd put an example of the http request showing the parameters here first before diving into `Context` objects etc, this feels like it goes from 0-60